### DISCUSSION TITLE
Span element inside a p element gets removed

### DISCUSSION THREAD

**Discussion by samyarkd** [ueberdosis/tiptap#3591] [General Questions]
Hi,

I give the editor HTML content to parse it, for example (I placed span elements in 3 positions): 
```Html
<p>Hello this is a test</p><p>d <span keyID="#key09"> </span> a</p><p><span keyID="#key09"> </span></p><p>s<span keyID="#key09"> </span>v</p><p>a</p>
```

And the rendered result in editor looks like this (The question marks are removed span elements got removed ones): 
```Html
<p>Hello this is a test</p><p>d **?** a</p><p>**?**</p><p>s<span keyID="#key09"> </span>v</p><p>a</p>
```

as you can see the only one who is still remained is the one who got text around it (not even space it should be only text.) i don’t want anything to be removed.

**Comment by tmikaeld** [comment]
We also hit this now, why is the span even removed? A span inside a p tag is valid html!
